tsconfig.json

```diff
@@ -0,0 +1,56 @@
+{
+  "compilerOptions": {
```

Comment:
Let's either—

* update to the latest version of ember-cli-typescript and re-run `ember generate ember-cli-typescript`
* or manually copy over [the blueprint's](https://github.com/typed-ember/ember-cli-typescript/blob/master/blueprint-files/ember-cli-typescript/tsconfig.json) `compilerOptions` for everything but `paths`

—that will give us a much better version of this `tsconfig.json`!